include/flatbuffers/flexbuffers.h

```diff
@@ -367,7 +367,11 @@ class Reference {
   bool IsNumeric() const { return IsIntOrUint() || IsFloat(); }
   bool IsString() const { return type_ == FBT_STRING; }
   bool IsKey() const { return type_ == FBT_KEY; }
+  bool IsTypedVectorElementType() const { return flexbuffers::IsTypedVectorElementType(type_); }
```

Comment:
can you put these in the opposite order? First `IsVector` etc. It is nice if the reader can go from most general to most specific for vectors.

Reply:
Ok will do that.. but IsAnyVector will still be at the end as it is using other apis.
So new order.
IsVector
IsTypedVector
IsFixedTypedVector
IsAnyVector
Seems fine?

Reply:
`IsTypedVectorElementType` is still in the wrong order?